docs/section19/19-12.md

```diff
@@ -0,0 +1,47 @@
+# Section 19 - Rule 19-8        
+**Schema Version:** 0.0.21  
+**Mandatory Rule:** True  
+**Rule ID:** 19-8         
+**Rule Description:** Demand control ventilation is modeled in the baseline design in systems with outdoor air capacity greater than 3000 cfm serving areas with an average occupant design capacity greater than 100 people per 1000 ft^2.     
+**Rule Assertion:** Options are Pass/Fail     
+**Appendix G Section:** Section G3.1.2.5 Excetion #1      
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each HeatingVentilationAirConditioningSystem Data Group  
+
+**Applicability Checks:** None  
+
+**Function Calls:**  
+1. get_hvac_zone_list_w_area()  
+
+
+## Rule Logic:  
+- Get dictionary with hvac systems and associated zones and areas: `hvac_zone_list_w_area_dict_b  = get_hvac_zone_list_w_area(B_RMR)`  
+- For each hvac_b system in B_RMR: `for hvac_b in B_RMR...Zone:`  
```

Comment:
Should this be:
`for hvac_b in hvac_zone_list_w_area_dict_b.keys()"?

Reply:
@weilixu This is the notation that Xing and I have been using when we want to check all of whatever it is in the baseline. We have been leaving it up to the RCT to decide how to do it. One approach is as you suggest so no problem to change it but just giving a heads up as to this is how the RDSs are generally written (which I am sure you already know! :))